apps/src/code-studio/components/progress/ScriptOverviewTopRow.jsx

```diff
@@ -87,6 +95,19 @@ const ScriptOverviewTopRow = React.createClass({
             assignmentName={scriptTitle}
           />
         )}
+        {!professionalLearningCourse && viewAs === ViewType.Teacher &&
+            resources.length > 0 &&
+          <div style={styles.dropdown}>
+            <DropdownButton
+              text={i18n.teacherResources()}
+              color={Button.ButtonColor.blue}
+            >
+              {resources.map(({type, link}, index) =>
+                <a key={index} href={link}>{stringForType[type]}</a>
```

Comment:
(These all just have hrefs of course, but my other usage of DropdownButton has onclick and href).